measurements/regard/README.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Regard
+emoji: 🤗
+colorFrom: green
+colorTo: purple
+sdk: gradio
+sdk_version: 3.0.2
+app_file: app.py
+pinned: false
+tags:
+- evaluate
+- measurement
+description: >-
+Regard aims to measure language polarity towards and social perceptions of a demographic (e.g. gender, race, sexual orientation).
+---
+
+# Measurement Card for Regard
+
+## Measurement Description
+
+The `regard` measurement returns the estimated language polarity towards and social perceptions of a demographic (e.g. gender, race, sexual orientation).
+
+It uses a model trained on labelled data from the paper ["The Woman Worked as a Babysitter: On Biases in Language Generation" (EMNLP 2019)](https://arxiv.org/abs/1909.01326)
+
+## How to Use
+
+This measurement requires a list of strings as input:
+
+```python
+>>> regard = evaluate.load("regard", module_type="measurement")
+>>> input_texts = ["these girls are silly", "these boys are lost"]
+>>> results = regard.compute(predictions=input_texts)
+```
+
+### Inputs
+- **predictions** (list of `str`): A list of prediction/candidate sentences
+- **aggregation** (`str`) (optional): determines the type of aggregation performed on the data.
+    If set to `None`, the scores for each sentences are returned.
+     Otherwise:
+        - `average` : returns the average regard for each category (negative, positive, neutral, other)
+        - `maximum`: returns the maximum regard for each category
```

Comment:
Is there precedent for reporting aggregations like this? I might be missing something — right now I'm unclear on what I am supposed to glean from a metric like "maximum regard for this set of sentences over the positive label" or "average regard for these sentences over the negative label". Aggregations made sense for metrics like toxicity ("how often are samples are marked as toxic" for toxicity probability and "what is the worst output which a model can do" for maximum toxicity), but I'm not sure that they make sense here as much. I think it would make more sense to aggregate over the group we're trying to measure regard for (e.g. "what is the level of regard for asian people vs. level of regard for white people"), though I recognize that's out-of-scope for the current format of the input data (we'd need groupings of "sentences about xyz subgroup" for this).

Reply:
right, that's what I put in the comment for this commit (see above) -- would it make sense for the default behavior to take two lists of strings instead of one?

Reply:
Ah yes thank you, I saw that then promptly forgot about it as I got deeper into the code, sorry! Yes I agree with you, I think it'd be optimal for the default behaviour to take two lists of strings where each list is a subgroup we'd like to measure against.